engine/lib/admin.php

```diff
@@ -196,7 +196,11 @@ function _elgg_create_notice_of_pending_upgrade($event, $type, $object) {
  */
 function _elgg_admin_init() {
 
-	elgg_register_event_handler('pagesetup', 'system', '_elgg_admin_pagesetup', 1000);
+	$url = elgg_get_simplecache_url('admin.css');
+	elgg_register_css('elgg.admin', $url);
+		
+	elgg_register_plugin_hook_handler('register', 'menu:admin_header', '_elgg_admin_header_menu');
```

Comment:
Just thinking out loud: I wonder if we should register all these (formally using elgg_register_menu_item) with a low priority, like 1. Consider the case where a 2.x plugin is also registering for this hook, and expects the menu items already to be present. It's more of an issue in the plugins, because their inits potentially get called after a 3rd party plugin.